<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsItem</class>
 <widget class="QWidget" name="SettingsItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Exclude directories:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Index update mode:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="editIndexDB"/>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="comboExtensions">
     <property name="toolTip">
      <string>Comma separated list of extensions. Each extension needs to start with a dot. Use a single dot to select files without extension.</string>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Index type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="editName"/>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="editExcludeDirectories">
     <property name="toolTip">
      <string>Comma separated list of strings. If a string is part of the directory being indexed it is exluded.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelExtensions">
     <property name="text">
      <string>Extensions:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="comboIndexUpdateMode">
     <property name="insertPolicy">
      <enum>QComboBox::NoInsert</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="labelIndexLocation">
     <property name="text">
      <string>Index location:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelDirectories">
     <property name="text">
      <string>Directory:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="buttonBrowseIndexLocation">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="editDirectories">
     <property name="toolTip">
      <string>Comma separated list of directories to index.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="buttonDirectory">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="comboIndexType"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>editDirectories</tabstop>
  <tabstop>buttonDirectory</tabstop>
  <tabstop>comboExtensions</tabstop>
  <tabstop>editExcludeDirectories</tabstop>
  <tabstop>comboIndexUpdateMode</tabstop>
  <tabstop>comboIndexType</tabstop>
  <tabstop>editIndexDB</tabstop>
  <tabstop>buttonBrowseIndexLocation</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonDirectory</sender>
   <signal>clicked()</signal>
   <receiver>SettingsItem</receiver>
   <slot>browseForDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBrowseIndexLocation</sender>
   <signal>clicked()</signal>
   <receiver>SettingsItem</receiver>
   <slot>browseForIndexDB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>browseForDirectory()</slot>
  <slot>browseForIndexDB()</slot>
  <slot>updateIndex()</slot>
 </slots>
</ui>
